Jint/Runtime/Debugger/DebugInformation.cs

```diff
@@ -1,16 +1,45 @@
 ﻿using System;
+using System.Collections;
 using System.Collections.Generic;
+using System.Linq;
+using Esprima;
 using Esprima.Ast;
 using Jint.Native;
 
 namespace Jint.Runtime.Debugger
 {
     public class DebugInformation : EventArgs
```

Comment:
we should probably seal this too

Reply:
I've gone ahead and also gotten rid of public setters, adding an `internal` constructor to `DebugInformation`. I'm not sure what the original reasoning behind `CurrentMemoryUsage` is - it's never set in the engine. But devtools protocol does have a `getHeapUsage` method, so might be useful in future - keeping it for now, and adding an XMLDoc about it not being useful now.